configs/paths.go

```diff
@@ -34,9 +34,9 @@ const (
 	BitriseSourceDirEnvKey = "BITRISE_SOURCE_DIR"
 	// BitriseDeployDirEnvKey ...
 	BitriseDeployDirEnvKey = "BITRISE_DEPLOY_DIR"
-	// BitriseTestDeployDirEnvKey ...
+	// BitriseTestDeployDirEnvKey is the root directory of test reports
 	BitriseTestDeployDirEnvKey = "BITRISE_TEST_DEPLOY_DIR"
-	// BitriseTestResultDirEnvKey ...
+	// BitriseTestResultDirEnvKey is a unique subdirectory in BITRISE_TEST_DEPLOY_DIR for each step run, steps should place test reports and attachments into this directory
```

Comment:
Could call it BitrisePerStepTestResultDirEnvKey to make it more descriptive when referencing